widgets/label.go

```diff
@@ -205,16 +205,16 @@ func getTextAlignment(alignment Alignment) text.Alignment {
 	}
 }
 
-func getLayoutAlignment(alignment Alignment) layout.Alignment {
-	switch alignment {
-	case AlignMiddle:
-		return layout.Middle
-	case AlignRight:
-		return layout.End
-	default:
-		return layout.Start
-	}
-}
+// func getLayoutAlignment(alignment Alignment) layout.Alignment {
+// 	switch alignment {
+// 	case AlignMiddle:
+// 		return layout.Middle
+// 	case AlignRight:
+// 		return layout.End
+// 	default:
+// 		return layout.Start
+// 	}
+// }
```

Comment:
Can you stop just commenting functions out with no reason? If it is not being used it should not be in the code base.